packages/medusa/src/models/line-item.ts

```diff
@@ -30,40 +30,60 @@ export class LineItem extends BaseEntity {
   @Column({ nullable: true })
   cart_id: string
 
-  @ManyToOne(() => Cart, (cart) => cart.items)
+  @ManyToOne(
+    () => Cart,
+    (cart) => cart.items
+  )
   @JoinColumn({ name: "cart_id" })
   cart: Cart
 
   @Index()
   @Column({ nullable: true })
   order_id: string
 
-  @ManyToOne(() => Order, (order) => order.items)
+  @ManyToOne(
```

Comment:
question: Did something change with the linter? is that linked to the prettier add to the conf? because I see a lot of lint fixing